features/ui/sidebar-navigation/sidebar-navigation.tsx

```diff
@@ -196,6 +199,7 @@ export function SidebarNavigation() {
               onClick={() => alert("Support")}
             />
             <CollapseMenuItem
+              isSidebarCollapsed={isSidebarCollapsed}
```

Comment:
This is exactly how it's supposed to be done. Only one thing: This component already has a `isCollapsed` prop a few lines down. So this new prop isn't required

Reply:
Thank you, probably went too fast on this one :). I made a new PR with code refactoring.